packages/terser/src/index.ts

```diff
@@ -1,24 +1,8 @@
-import { NormalizedOutputOptions, RenderedChunk } from 'rollup';
-import { minify, MinifyOptions } from 'terser';
+import { runWorker } from './worker';
+import terser from './module';
 
-export default function terser(options?: MinifyOptions) {
-  return {
-    name: 'terser',
+runWorker();
```

Comment:
Does this need to be called here? It seems like it'll just return since it's on the main thread.

Reply:
Yes, it is required, due the fact that the files are bundled to one file. 
Thererfore, we must always expect, that the bundled file is called as worker.